aderyn_core/src/ast/contracts.rs

```diff
@@ -32,45 +32,6 @@ pub enum ContractDefinitionNode {
     UserDefinedValueTypeDefinition(UserDefinedValueTypeDefinition),
 }
 
-impl<'de> Deserialize<'de> for ContractDefinitionNode {
```

Comment:
Since we're now tagging the enum with `nodeType` we can remove this custom deserializing logic